### ISSUE TITLE
Test failure in cohen_acceleration_test with Apple Clang 12.0 and C++17

### ISSUE THREAD

**Issue by evanmiller** [boostorg/math#467]
Full log:

https://github.com/evanmiller/math/runs/1597837166

Relevant portion (starts at line 488 above):

```
testing.capture-output ../../../bin.v2/libs/math/test/cohen_acceleration_test.test/clang-darwin-12.0/debug/cxxstd-17-iso/link-static/threading-multi/visibility-hidden/cohen_acceleration_test.run
====== BEGIN OUTPUT ======
Error at cohen_acceleration_test.cpp:test_divergent:68:
  ULP distance in long double precision is 134, which exceeds 80, error/ulps  = 1.67.
  Expected: +0.225791352644727432374 = +0xe.735d92d5098ae24p-6
  Computed: +0.225791352644727430558 = +0xe.735d92d5098ad9ep-6
  Mollified relative error: +1.81603863891321992696e-18
Error count: 1, total ulp distance = 134

EXIT STATUS: 1
====== END OUTPUT ======
```

**Comment by jzmaddock** [comment]
@NAThompson

**Comment by NAThompson** [comment]
There's no problem just increasing the tolerance on this one to 134; 80 ULPs was just empirical anyway-we have no theoretical reason to expect 80 ULP accuracy.

**Comment by NAThompson** [comment]
Should be fixed in 5fc9214.
